libs/ledger-live-common/src/families/cosmos/js-prepareTransaction.ts

```diff
@@ -42,58 +39,54 @@ export const calculateFees: CacheRes<
       transaction.sourceValidator ? transaction.sourceValidator : ""
     }`,
   {
-    ttl: 1000 * 60, // 60 sec
+    ttl: 1000 * 10, // 10 sec
   },
 );
 
 export const getEstimatedFees = async (
   account: CosmosAccount,
   transaction: Transaction,
-): Promise<{ estimatedFees: BigNumber; estimatedGas: BigNumber }> => {
-  const cosmosCurrency = cryptoFactory(account.currency.id);
-  let estimatedGas = new BigNumber(cosmosCurrency.defaultGas);
+): Promise<{
+  gasWanted: BigNumber;
+  gasWantedFees: BigNumber;
+}> => {
+  const chainInstance = cryptoFactory(account.currency.id);
+  let gasWanted = new BigNumber(chainInstance.defaultGas);
```

Comment:
no need to give an initial value to gasWanted.
line 83, you can just write:
const gasWanted = gasUsed.times(getEnv("COSMOS_GAS_AMPLIFIER")).integerValue(BigNumber.ROUND_CEIL);

Reply:
indeed, done